src/main/java/ReadFile.java

```
@@ -0,0 +1,62 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReadFile {
+    public static void readFileToArrayList() throws IOException, RoleyPoleyException {
+        try {
+            Path myPath = Paths.get("./src/main/java/RoleyPoleyData.txt");
+            List<String> taskList = Files.readAllLines(myPath, StandardCharsets.UTF_8);
+            for(String line : taskList) {
+                if (line.trim().isEmpty()) {
+                    continue;
+                } else {
+                    try {
+                        convertTask(line);
+                    } catch (RoleyPoleyException e) {
+
```

avoid empty catch blocks